careers/blocks/slider/slider.js

```diff
@@ -0,0 +1,218 @@
+import { buildEllipsis } from '../../scripts/scripts.js';
+
+const DEFAULT_CONFIG = Object.freeze({
+  'visible-slides': 3,
+  maxlines: 3,
```

Comment:
If we remove the `buildEllispsis`, we call also simplify this...

Reply:
Done.